bundler.d/assets.rb

```diff
@@ -1,7 +1,7 @@
 group :assets do
   gem 'jquery-ui-rails', '~> 6.0'
   gem 'patternfly-sass', '~> 3.59.4'
-  gem 'gettext_i18n_rails_js', '~> 1.0'
+  gem 'gettext_i18n_rails_js', '~> 1.0', git: 'https://github.com/webhippie/gettext_i18n_rails_js'
```

Comment:
can we convince the author to release?

a git URL is nothing we can use in packaging, so prod isntalls would still suffer from the issue :(

Reply:
we don't need it in prod.
and the last comment in the issue was in 2019, so I'm not sure about that https://github.com/webhippie/gettext_i18n_rails_js/issues/41

Reply:
don't we need it to build assets?

Reply:
Not sure how gems and packaging works, but this is only used in extracting translations, so it probably shouldn't be used in prod

Reply:
Okay. I'll try to dig into that tomorrow.

Thanks!

Reply:
So I actually tried, and it builds fine w/o that gem being present, if I also remove it from the `asset` group here.

I guess adding the `git` link here is fine, it shouldn't confuse bundler when we're building. And long term we could even drop this dep from this group completelly.

Reply:
So I guess the author has no time to maintain, so either we help, or find other ways like native JS extraction e.g. https://www.npmjs.com/package/gettext-extractor and combine manually.

Reply:
Given this is a trivial change and a devel-time only dependency, I am cool if we merge this PR to make our `.po` files more correct and later search for a nicer replacement of this.

Just for completeness of my understanding:
- This GEM does have other uses besides the Rake task for extraction, but we (Foreman) only use the Rake task
- The rake task is something we run when we do a release, to extract new strings
- The replacement of translated strings in the UI happens by some other means and is untouched in this PR

If all of the above is true, then you have my ACK for this.

Reply:
@MariaAga I think all of the above is true, but could you check as well? :)

Reply:
Yup, from what I know these are all true :)

Reply:
I'd prefer to pin. In a `Gemfile` you can use `ref: 'COMMIT'`. You can also drop `'~> 1.0'` if you point to git.

Reply:
Added a ref and removed the version